openshift-cert-manager-operator/operator/base/operator-group.yaml

```diff
@@ -1,9 +1,8 @@
 apiVersion: operators.coreos.com/v1
 kind: OperatorGroup
 metadata:
-  name: cert-manager-operator
+  name: openshift-cert-manager-operator-group
```

Comment:
I don't think it is a requirement, but I think normally the operator groups name is the same as the namespace.  I would prefer to keep it the same as whatever the GUI creates when installing through the UI so if someone installs via the UI and then later adds the catalog item into gitops, it doesn't cause a second operator group to be created.

Can you confirm what the correct operatorgroup name should be?

Reply:
I agree with aligning the name of the operator group with the packagemanifest name, that might also be the namespace name... Either way we can remove the `-group` because it's redundant. This should be removed globally and in the script used to generate operators. I can do that in another PR.